google-oidc/app.py

```
@@ -1,21 +1,20 @@
-oauth.register(
-    name='google',
-    server_metadata_url=CONF_URL,
-    client_kwargs={
-        'scope': 'openid email profile'
-    }
-)
+GOOGLE_ISSUER = "https://accounts.google.com"
+DISCOVERY_URL = f"{GOOGLE_ISSUER}/.well-known/openid-configuration"
+
+def get_google_cfg():
+    return requests.get(DISCOVERY_URL).json()
```

Ruff (0.14.13)

Probable use of `requests` call without timeout

(S113)